.eslintrc.js

```diff
@@ -13,6 +13,9 @@ module.exports = {
   rules: {
     'no-console': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
     'no-debugger': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
+    // if the line goes more than 120 , it will be an error
+    'max-len': ['error', 120],
+    'no-tabs': 0,
```

Comment:
what is no-tabs rule? There should be no tabs, spaces only. This should definitely be removed. Mixed tabs and spaces is a nightmare.

Reply:
I had an issue with eslintter, so it was an answer I found on the web?

Reply:
What was the issue? There should be no issues with spaces. If you are using table, please switch to spaces. Most programmer's editors have options to insert spaces instead of tabs. This rule should definitely stat enabled and give an error.

Reply:
I just commented line 18